src/components/AddNewBook.js

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
+
+const AddNewBook = () => {
+  const dispatch = useDispatch();
+
+  const title = useRef();
+  const author = useRef();
+
+  const handleAddBook = (e) => {
+    e.preventDefault();
+    dispatch(addBook({
+      item_id: Date.now().toString(),
+      title: title.current.value,
+      author: author.current.value,
+    }));
+    title.current.value = '';
+    author.current.value = '';
+    title.current.focus();
+  };
+
+  return (
+    <section>
+      <h2>ADD NEW BOOK</h2>
+      <form onSubmit={handleAddBook}>
```

Comment:
- [ ] Kindly implement this handler in the  `<Button>` but in the button case, it should include an event handler to add a book to the books array (with attributes title and author)

### Sample:
```
<Button onClick={() => dispatch(addBook({ title, author }))}>
        Add book
      </Button>
```

## Requirements:
![image](https://github.com/astroboyReloaded/Bookstore/assets/50990925/4f008ab2-67d2-47e6-93a5-5cabdf769688)

Reply:
Thank you very much for your comments, @Shedrack-Sunday!
I do have a question, does it have to have a `type="submit"`, or can it be a `type="button"`?

Reply:
@astroboyReloaded, You have a similar implementation the `Remove Button`. I'll advise you to follow the same pattern. It's going to be a button component with an event handler.